test/correct-filename.test.js

```diff
@@ -1,73 +1,70 @@
-var should = require("should");
-var path = require("path");
-var fileLoader = require("../");
+const fileLoader = require('../');
```

Comment:
Looking at some of the libs, maybe a `test` folder is going to be the cleaner way to do this. Jest is smart enough to handle the preprocessing regardless of where the tests reside.